docs/user-guide/configuration/workflow.md

```diff
@@ -157,6 +159,30 @@ jobs:
             - echo: echo hi
 ```
 
+## Blocked By
+To have your job blocked by another job, you can use `blockedBy`. It has the same format as `requires`, except it does not accept values like `~commit` or `~pr`.
+Note:
+- To prevent race conditions, a job is always blocked by itself. That means the same job cannot have 2 instances of builds running at the same time.
+- This feature is only available if your cluster admin configured to use `executor-queue`. Please double check with your cluster admin whether it is supported.
+
+#### Example
+In the following example, `job2` is blocked by `job1` or `sd@456:publish`. If `job1` or `sd@456:publish` is running and `job2` is triggered, `job2` will be put back into the queue.
```

Comment:
Everything in the blockedBy list is an OR, right?

If `job1` is running, `job2` will be re-enqueue. Or if `sd@456:publish` is running, `job2` will re-enqueue.

Reply:
Addressing this issue in this comment: https://github.com/screwdriver-cd/screwdriver/issues/653#issuecomment-392997990